resteasy-core/src/main/java/org/jboss/resteasy/core/ContextParameterInjector.java

```diff
@@ -38,13 +37,39 @@
 @SuppressWarnings("unchecked")
 public class ContextParameterInjector implements ValueInjector
 {
+   private static Constructor<?> constructor;
+
    private Class<?> rawType;
    private Class<?> proxy;
    private ResteasyProviderFactory factory;
    private Type genericType;
    private Annotation[] annotations;
    private volatile boolean outputStreamWasWritten = false;
 
+   static
+   {
+      try
+      {
+         Class.forName("javax.servlet.http.HttpServletResponse");
+         Class<?> clazz = Class.forName("org.jboss.resteasy.core.ContextServletOutputStream");
+         constructor = clazz.getConstructor(ContextParameterInjector.class, OutputStream.class);
```

Comment:
Since this is now package-private this needs to be `clazz.getDeclaredConstructor(ContextParameterInjector.class, OutputStream.class)`.

Reply:
Done